docs/source/release_roadmap.org

```diff
@@ -0,0 +1,363 @@
+#+TITLE: Recommendations regarding the release roadmap of Iroha v2
+#+AUTHOR: Aleksandr Petrosyan
+#+DATE: 2023-08-13
+
+* State of the project
+
+Iroha v2 is a capable ledger with notable subroutines that allow it to
+be deployed successfully as a private blockchain ledger:
+1. It has relatively good performance characteristics regarding
+   throughput (post RC9).
+2. It has good performance characteristics regarding transaction
+   finality and latency (post RC19 also good query latency).
+3. It has support for Turing complete permission validation, which
+   allows Iroha to enforce arbitrarily complex invariants on-chain. 
+4. It has support for a small interpreted VM called the Iroha Special
```

Comment:
Is it correct to call it an "interpreted VM"

Reply:
Clarified.